pkgs/development/ocaml-modules/macaddr/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, buildDunePackage, ocaml
+, ppx_sexp_conv, sexplib
+}:
+
+buildDunePackage rec {
+  pname = "macaddr";
+  version = "3.1.0";
+
+  minimumOCamlVersion = "4.04";
+
+  src = fetchFromGitHub {
+    owner = "mirage";
+    repo = "ocaml-ipaddr";
+    rev = "v${version}";
+    sha256 = "0g7wgfn06z7fhdk4hlblf7b5k2ws56z0l73gk7189p5wvrmbavvx";
+  };
+
+  buildInputs = [ sexplib ];
```

Comment:
Why is `sexplib` needed?

Reply:
Good catch, looks like it isn't - I don't quite remember what made me add it, but it was probably a misguided attempt to fix the following error, which I get if I `ppx_sexp_conv` isn't a dependency  
```
building '/nix/store/vm9lr1n0cj93iqc0xx5rl3plxlwnw9qp-ocaml4.07.1-macaddr-3.1.0.drv'...
unpacking sources
unpacking source archive /nix/store/zljnp91y7ig1av99d3l0kiacpn39b3ab-v3.1.0.tar.gz
source root is ocaml-ipaddr-3.1.0
setting SOURCE_DATE_EPOCH to timestamp 1551633159 of file ocaml-ipaddr-3.1.0/macaddr.opam
patching sources
configuring
no configure script, doing nothing
building
File "lib/dune", line 22, characters 20-28:
22 |  (libraries macaddr sexplib0))
                         ^^^^^^^^
Error: Library "sexplib0" not found.
```

Reply:
The `sexplib0` library is packaged under the `sexplib0` attribute. It is different than `sexplib`.